src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx

```diff
@@ -48,7 +56,34 @@ class DowntimeNotification extends React.Component {
     if (this.props.shouldSendRequest) this.props.getScheduledDowntime();
   }
 
+  renderGlobalDowntimeOverride = () => {
+    const appType = Object.values(VA_FORM_IDS).includes(this.props.appTitle)
+      ? 'form'
+      : 'tool';
+
+    const downtimeEnd = formatDowntime(scheduledDowntimeWindow.downtimeEnd);
+
+    return (
+      <AlertBox
+        className="vads-u-margin-bottom--4"
+        headline={`This ${appType} is down for maintenance`}
+        isVisible
+        status="warning"
+      >
+        <p>
+          We’re making some updates to this {appType}. We’re sorry it’s not
+          working right now and we hope to be finished by {downtimeEnd}. Please
+          check back soon.
+        </p>
+      </AlertBox>
+    );
+  };
+
   render() {
+    if (isGlobalDowntimeInProgress()) {
+      return this.renderGlobalDowntimeOverride();
```

Comment:
Sanity checking my understanding: If we pass in a `render` prop to the `DowntimeNotification`, this message will take precedent if there's a global downtime, right?

Reply:
Yup, this occurs before the `render` prop is taken into consideration (right below this).